### PR TITLE
Nep 42 news details

### PR DESCRIPTION
**Pull request by shrijan-siris**

### CODE REVIEW

File: src/frame/app-container.js [hunk 1]
```diff
@@ -4,11 +4,13 @@ import Home from '../home/index.js'
...
 import BottomNavigation from '../home/components/bottom-navigation/index'
+import ArticleDetailScreen from '../home/components/article-detail.screen.js'
 
 const AppNavigator = createStackNavigator(
 	{
 		Home: { screen: Home },
 		Article: { screen: ArticleScreen },
+		ArticleDetail: { screen: ArticleDetailScreen },
 		Twitter: { screen: TwitterScreen },
 		BottomNavigation: { screen: BottomNavigation }
```
Comment (syuraj) [inline src/frame/app-container.js]: I think we can probably remove `BottomNavigation ` from here

**Comment by syuraj** [comment]
Can you clean up this PR , probably rebase with master to show only new commits? 
Right now, it's hard to read as it is showing old commits too.
Also, can you include a screenshot of ArticleDetail screen?

@shrijan-siris

File: android/build.gradle [hunk 2]
```diff
@@ -13,7 +13,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.0'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```
[the remark this answers is not in the document]
Comment (shrijan-siris) [reply android/build.gradle]: This is the one you have downgraded before

File: src/frame/app-container.js (hunk 1 above)
Comment (shrijan-siris) [reply src/frame/app-container.js]: yes we can

**Pull request closed without merging**